covid19-etl/utils/metadata_helper.py

```diff
@@ -203,7 +162,27 @@ def batch_submit_records(self):
 
         self.records_to_submit = []
 
-    async def query_node_data(self, query_string):
+    def query_peregrine(self, query_string):
+        url = f"{self.base_url}/api/v0/submission/graphql"
+        response = requests.post(
+            url,
+            json={"query": query_string, "variables": None},
+            headers=self.headers,
+        )
+        try:
+            response.raise_for_status()
+        except Exception:
+            print(
+                f"Unable to query Peregrine.\nQuery: {query_string}\nVariables: {variables}"
+            )
+            raise
+        try:
+            return response.json()
+        except:
+            print(f"Peregrine did not return JSON: {response.text}")
+            raise
+
+    async def query_peregrine_async(self, query_string):
```

Comment:
Can you change the name to `async_query_peregrine` so that it is consistent to other async function names?